modules/core/src/lib/select-loader.js

```diff
@@ -95,6 +97,16 @@ function findLoaderByExtension(loaders, extension) {
   return null;
 }
 
+// data should be a Response object
+function findLoaderByContentType(loaders, data) {
+  if (!data || !data.headers || !(data.headers instanceof Headers) || !isFetchResponse(data)) {
```

Comment:
It's generally not recommended to use `instanceof`, since it relies on the existence of a global.

It's easier to check `isFetchResponse` then `data.headers && data.headers.get(...)`